oidc.go

```diff
@@ -158,6 +162,27 @@ func (p *Provider) Endpoint() oauth2.Endpoint {
 	return oauth2.Endpoint{AuthURL: p.authURL, TokenURL: p.tokenURL}
 }
 
+// LogoutURL returns the logout endpoints for the given provider.
+// See: https://openid.net/specs/openid-connect-session-1_0.html#RPLogout
+func (p *Provider) LogoutURL(idTokenHint string, redirectURL string, state string) string {
```

Comment:
Where do these values come from? E.g. how would a user get the idTokenHint and redirectURL?

Reply:
For `idTokenHint` I am using the token value, which works on Azure AD OpenID Connect. The `redirectURL` I am getting off of the request `url` query parameter, but sources could differ. It must be registered as a valid callback url, just like a login url. This callback makes sure that the application session is ended and can be registered separately as a logout url in Azure so that it calls all registered logout url's when a user has signed off, effectively implementing single signoff.